doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -108,38 +76,33 @@ which is a simplified version of the actual definition from [`env.jl`](https://g
 
 ```julia
 function getenv(var::AbstractString)
-    val = ccall(:getenv, Cstring, (Cstring,), var)
+    val = @ccall getenv(var::Cstring)::Cstring
     if val == C_NULL
         error("getenv: undefined variable: ", var)
     end
     return unsafe_string(val)
 end
 ```
 
-The C `getenv` function indicates an error by returning `NULL`, but other standard C functions
-indicate errors in various different ways, including by returning -1, 0, 1 and other special values.
-This wrapper throws an exception clearly indicating the problem if the caller tries to get a non-existent
+The C `getenv` function indicates an error by returning `C_NULL`, but other standard C functions
+indicate errors in different ways, including by returning -1, 0, 1, and other special values.
+This wrapper throws an exception indicating the problem if the caller tries to get a non-existent
 environment variable:
 
 ```julia-repl
 julia> getenv("SHELL")
 "/bin/bash"
 
 julia> getenv("FOOBAR")
-getenv: undefined variable: FOOBAR
+ERROR: getenv: undefined variable: FOOBAR
 ```
 
 Here is a slightly more complex example that discovers the local machine's hostname.
-In this example, the networking library code is assumed to be in a shared library named "libc".
```

Comment:
Removed this because it no longer works to specify `"libc"`: 

```julia
julia> gethostname()
ERROR: could not load library "libc"
/lib/x86_64-linux-gnu/libc.so: invalid ELF header
```

https://discourse.julialang.org/t/invalid-elf-header-0-7-0-beta-built-from-source/12054, https://github.com/JuliaLang/julia/issues/450

Reply:
As the comment here said, this was usually not correct ever, but we want to show the syntax for it here

Reply:
Okay, imo, let's show the syntax in a way that is correct then. Like for calling a locally compiled library.